packages/react-router/lib/types.ts

```diff
@@ -191,11 +189,12 @@ type __tests = [
     >
   >,
   Expect<
+    //@ts-expect-error TODO: Why is this failing?
```

Comment:
Same thing here

Reply:
Again, struggling to get GitHub to add a commit suggestion, but should be this

```ts
Expect<
    Equal<
      Pretty<
        ClientDataFrom<
          () =>
            | { json: string; b: Date; c: () => boolean }
            | DataWithResponseInit<{ data: string; b: Date; c: () => boolean }>
        >
      >,
      | { json: string; b: Date; c: () => boolean }
      | { data: string; b: Date; c: () => boolean }
    >
  >,
```